### PR TITLE
2899 closing mobile filtering

### LINKED ISSUE #2899: mobile filters cannot close. there's no 'X'
demo uwazi io_(Galaxy S5)

### PR DESCRIPTION
**Pull request by mfacar**
fixes #2899

This PR includes:
- [ ] X button on filters sidepanel on mobile view from library
- [ ]  Replace search icon with filter on mobile header

PR checklist:
- [ ] Update READ.me ?
- [ ] Update API documentation ?

QA checklist:
- [ ] Smoke test the functionality described in the issue
- [ ] Test for side effects
- [ ] UI responsiveness
- [ ] Cross browser testing
- [ ] Code review

### CODE REVIEW

**Comment by konzz** [comment]
LGTM, resolve the conflicts

**Comment by mfacar** [comment]
> LGTM, resolve the conflicts

done :)

**Comment by RafaPolit** [comment]
Please, check the breakpoint:

Sidepanel breakpoint from collapsible to always-visible is 1023px, not 769px.  I don't know how this value became the standard, but you cannot close the sidepanel from 769 to 1023 with this approach.

Also, this is not working on iOS.

The side bar is not positioned on top of navbar, but rather underneath, making it impossible to click on the X.

Also, using in mobile iOS, the "close" arrow is correctly present on the header replacing the "search" button (now the funnel).  So, either:
- ensure that the side panel is on top of the nav bar, or
- move the sidepanel on mobile below the nav bar

File: app/react/App/scss/layout/_sidepanel.scss [hunk 1]
```diff
@@ -20,6 +20,11 @@ $c-sidebar: $c-white;
   &.is-hidden {
     right: -100%;
   }
+
+  .only-mobile { display: inline-block; };
+  @media (min-width: 769px) {
```
Comment (RafaPolit) [inline app/react/App/scss/layout/_sidepanel.scss:25]: This breakpoint leaves from 769 to 1023 without the option to close the sidepanel.

**Review by RafaPolit — CHANGES_REQUESTED** [review]

**Comment by mfacar** [comment]
> Also, using in mobile iOS, the "close" arrow was correctly present on the header replacing the "search" button (now the funnel), which is no longer the case. So, either:
> 
> * ensure that the side panel is on top of the nav bar, or
> * move the sidepanel on mobile below the nav bar

I have applied the first option, iOS evidenced a problem with the overflow of the class library-viewer, it's fixed now.

**Comment by RafaPolit** [comment]
LGTM

**Pull request merged**